src/main/java/io/jenkins/plugins/gitlabbranchsource/GitLabSCMSource.java

```diff
@@ -720,10 +719,13 @@ public void afterSave() {
         GitLabHookCreator.register(this, webhookMode, systemhookMode);
     }
 
-    public PersonalAccessToken credentials() {
-        return CredentialsMatchers.firstOrNull(lookupCredentials(PersonalAccessToken.class, getOwner(),
-                Jenkins.getAuthentication(), fromUri(getServerUrlFromName(serverName)).build()),
-                GitLabServer.CREDENTIALS_MATCHER);
+    public StandardCredentials credentials() {
```

Comment:
Changing a signature of a method is not encourage. Especially this is a public method, other users might already are using it in their own plugin.

Reply:
@conan17 You can add deprecate the old method and add your new one. For ref see how it is done in https://github.com/jenkinsci/github-branch-source-plugin/blob/master/src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMSource.java#L667

Reply:
I kept this part of the code, which is consistent with the original